team-01/src/main/java/ubc/cosc322/MonteCarloPlayer.java

```diff
@@ -213,9 +214,9 @@ public AmazonsAction getAction() {
 
 		private double getUCB() {
 			// EXPLORATION_FACTOR = constant defined at the top of the class.
-			// If we hit 0, then the unvisited node should return infinity.
+			// If we hit 0, then the unvisited node should return the exploration factor.
 			if (this.getVisits() == 0) {
-				return Double.MAX_VALUE;
+				return EXPLORATION_FACTOR;
```

Comment:
This contradicts my primary source for the MCTS algorithm (John Levine's video linked at the top). I would love to read your source(s) which state this as a possible alternative UCB calculation.

Reply:
I think you're technically right here, when I was doing my initial implementation for the research steps I was looking at a lot of different implementations for different games, and a few seemed to use this method, but a standard ucb calculation will return infinity. I changed this because returning infinity throws an error, and returning this gave me a working player. We can change this if we want, but it works and we need a working agent for Monday, so maybe not a priority just yet